templates/feed-podcast.php

```diff
@@ -58,7 +58,7 @@
 	<channel>
 		<title><?php echo esc_html( $title ); ?></title>
 		<atom:link href="<?php self_link(); ?>" rel="self" type="application/rss+xml"/>
-		<link><?php echo esc_url( apply_filters( 'ssp_feed_channel_link_tag', $ss_podcasting->home_url, $podcast_series ) ) ?></link>
+		<link><?php echo $feed_link ?></link>
```

Comment:
W3 Validator wants that the artwork lead to the episodes list. Not to the the main RUL (domain) as it is now on 2.23.0.